apple/internal/aspects/resource_aspect.bzl

```diff
@@ -131,7 +131,18 @@ def _apple_resource_aspect_impl(target, ctx):
     # The name of the bundle directory to place resources within, if required.
     bundle_name = None
 
-    if ctx.rule.kind == "objc_library":
+    if ctx.rule.kind == "cc_library":
+        collect_args["res_attrs"] = ["data"]
+
+        # Only set cc_library targets as owners if they have srcs or deps. This
+        # treats cc_library targets without sources as resource aggregators.
+        if ctx.rule.attr.srcs or ctx.rule.attr.deps:
+            owner = str(ctx.label)
+
+    elif ctx.rule.kind == "cc_import":
```

Comment:
Could consolidate this with `"objc_import"` with an `in` check?

Reply:
Updated